### PR TITLE
Country was not showing in Client's address page in issue 

### PR DESCRIPTION
**Pull request by elirehema**
## Description
Resolved the countries selection problem by pointing out the list from empty list stateOptions to countryOptions

## Related issues and discussion
#3019

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [x] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Review by ShruthiRajaram — CHANGES_REQUESTED** [review]
Changing state to country is not the right way.  If I am not wrong change the ng-show="country" to ng-show="countryId" in addressForm html page. You will be able to fix the issue.

**Comment by elirehema** [comment]
@ShruthiRajaram any reason in short if possible,  while working on your request.

**Comment by ShruthiRajaram** [comment]
@elirehema kindly check the previous comment. You will be able to solve the issue with that.

**Comment by elirehema** [comment]
Okay @ShruthiRajaram am working on it as soon as possible.

**Comment by bauripalash** [comment]
@elirehema Any Update?

**Comment by elirehema** [comment]
@ShruthiRajaram @bauripalash  I have resolved it and and updated the PR

**Review by bauripalash — CHANGES_REQUESTED** [review]
Can you please add screenshots?

**Comment by elirehema** [comment]
@bauripalash check out on my PR, you can see the changes. Resolved as requested by @ShruthiRajaram

File: app/views/administration/AddressForm.html [hunk 1]
```diff
@@ -92,16 +92,16 @@
                     <label>{{ 'label.input.state_province' | translate }}</label>
                 </div>
                 <div class="col-sm-3" >
-                    <select chosen="stateOptions" id="stateOption" ng-model="formData.stateProvinceId"  class="form-control"
-                            ng-options="stateOption.id as stateOption.name for stateOption in stateOptions track by stateOption.id"
+                    <select chosen="countryOptions" id="stateOption" ng-model="formData.stateProvinceId"  class="form-control"
+                            ng-options="countryOption.id as countryOption.name for countryOption in countryOptions track by countryOption.id"
```
Comment (ShruthiRajaram) [inline app/views/administration/AddressForm.html]: these changes are not correct. Do not change the state field to country.

File: app/views/administration/AddressForm.html [hunk 2]
```diff
@@ -92,16 +92,16 @@
...
                             value="formData.state_province_id"></select>
                 </div>
             </div>
             <div>
 
             </div>
 
-            <div class="form-group" ng-show="country">
+            <div class="form-group" ng-show="countryId">
```
Comment (ShruthiRajaram) [inline app/views/administration/AddressForm.html:104]: Only this change is required

**Review by ShruthiRajaram — CHANGES_REQUESTED** [review]

**Review by ShruthiRajaram — APPROVED** [review]

**Pull request merged**